core/egov-localization/src/main/java/org/egov/persistence/repository/MessageRepository.java

```diff
@@ -83,7 +82,7 @@ public int getBatchSize() {
 		}
 		logger.info("Status of Insert is : " + values.length);
 		if(values.length > 0){
-			Tenant tenant = new Tenant(createMessagesRequest.getTenantId());
+			// Tenant tenant = new Tenant(tenantId);
 			// messageCacheRepository.bustCacheEntry(createMessagesRequest.getLocale(), tenant);
```

Comment:
@darshannagesh Leave it uncommented. I would anyways have to enhance the cache bust logic.